test/ng/compileSpec.js

```diff
@@ -4244,6 +4244,22 @@ describe('$compile', function() {
       if (!/chrome/i.test(navigator.userAgent)) return;
```

Comment:
Actually, it is not so easy @lgalfaso since the latest Safari (9.0.2) is even worse for `toString` on a a class instance.